modules/specialized-guides/pages/salt/salt-ssh.adoc

```diff
@@ -27,8 +27,16 @@ Use this method for in-place firewall environments that block HTTP or HTTPS conn
 As with all Salt calls, {productname} invokes `salt-ssh` via the `salt-api`.
 
 Salt SSH relies on a roster to obtain details such as hostname, ports, and the SSH parameters of a client.
-{productname} keeps these details in the database and makes them available to Salt by generating a temporary roster file for each Salt SSH call.
-The location of the temporary roster file is supplied to `salt-ssh` using the `--roster-file= option`.
+{productname} keeps these details in the database and makes them available to Salt by using `uyuni` roster module or by generating a temporary roster file on bootstrapping new clients with {webui} using Salt SSH call.
+The location of the temporary roster file is supplied to `salt-ssh` using the [option]``roster-file`` option.
+For the registered clients [option]``roster`` set to `uyuni` is used instead to get the roster from the database with `uyuni` salt roster module.
+
+[IMPORTANT]
+====
+It is not recommended to use [command]``salt-ssh`` as the call will be processed with current user permissions and in case of calling it with `root` user can cause permission issues for the files on processing next calls with `salt-api`.
+Use [command]``mgr-salt-ssh`` instead because it is changing effective user to `salt` on each call.
```

Comment:
```suggestion
Use [command]``mgr-salt-ssh``, which changes the effective user to [literal]``salt``and avoids file permission issues.
```

Reply:
```suggestion
Use [command]``mgr-salt-ssh``, which changes the effective user to [literal]``salt``and avoids file permission issues. In addition it sets [option]``roster`` to [literal]``uyuni``, if neither [option]``roster`` nor [option]``roster-file`` specified, to allow using {productname} database as a source of roster data for the registered Salt SSH clients.
```

Reply:
I thought that this "important" section should be kept short and direct. Everything you wrote is technically correct, I just wonder if it could distract from the message `use mgr-salt-ssh` due to its length.

Reply:
I see @agraul point here. I would prefer something short and effective sentence, then we can go and add the technical detail in a differente sentence maybe.